lib/matplotlib/rcsetup.py

```diff
@@ -767,6 +759,17 @@ def validate_cycler(s):
     else:
         raise ValueError("object was not a string or Cycler instance: %s" % s)
 
+    unknowns = cycler_inst.keys - (set(_prop_validators.keys()) |
+                                   set(_prop_aliases.keys()))
+    if unknowns:
+        raise ValueError("Unknown artist properties: %s" % unknowns)
+
+    # Not a full validation, but it'll at least normalize property names
+    # A fuller validation would require v0.10 of cycler.
+    for prop in cycler_inst.keys:
+        norm_prop = _prop_aliases.get(prop, prop)
+        cycler_inst.change_key(prop, norm_prop)
```

Comment:
maybe we should do this in two steps?  One explicit check of no aliased keys and then one normalization step we know will work?

I like that error message

Worried I am (re)painting the bikeshed with this review.